### ISSUE TITLE
Blacklist MineColonies for default conversion

### ISSUE THREAD

**Issue by sandalle** [MattDahEpic/AutoOreDictConvert1.8#20]
Hi there,

With autooredictconv-1.16.4-16.1.5.jar for Minecraft 1.16.5 and Forge 31.0.14 (haven't tried earlier ones), every time I add an ore with _/odc add_, it also adds five MineColonies (I also have that mod) entries, which are kind of like the `forge:ingots` in that they apply to every ingot, so when I put ingots into my inventory they all get converted to whatever I just added (a la https://github.com/MattDahEpic/AutoOreDictConvert/issues/11 ).

I'd like these `minecolonies:` entries to be blacklisted by default, if possible, so that only the `forge:ingots/lead tag` that I want gets added and no cleanup is necessary:
```
[21:51:49] [Server thread/INFO] [minecraft/DedicatedServer]: [sandalle: Added janoeo:lead_ingot as the default conversion for the minecolonies:sawmill_ingredient_excluded tag]
[21:51:49] [Server thread/INFO] [minecraft/DedicatedServer]: [sandalle: Added janoeo:lead_ingot as the default conversion for the minecolonies:blacksmith_product tag]
[21:51:49] [Server thread/INFO] [minecraft/DedicatedServer]: [sandalle: Added janoeo:lead_ingot as the default conversion for the minecolonies:blacksmith_ingredient tag]
[21:51:49] [Server thread/INFO] [minecraft/DedicatedServer]: [sandalle: Added janoeo:lead_ingot as the default conversion for the minecolonies:reduceable_product_excluded tag]
[21:51:49] [Server thread/INFO] [minecraft/DedicatedServer]: [sandalle: Added janoeo:lead_ingot as the default conversion for the forge:ingots/lead tag]
[21:51:49] [Server thread/INFO] [minecraft/DedicatedServer]: [sandalle: Added janoeo:lead_ingot as the default conversion for the minecolonies:reduceable_ingredient tag]
```

Here are the entries after adding the `janoeo:lead_ingot` that I was holding in my hand (I already cleaned up the other `janoeo` entries I added then decided to file this ;) ):
```
> odc list
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: The configured conversions are:
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: Tag Conversions
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - minecolonies:reduceable_product_excluded: janoeo:lead_ingot
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - forge:ingots/zinc: janoeo:zinc_ingot
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - minecolonies:sawmill_ingredient_excluded: janoeo:lead_ingot
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - forge:ores/lapis: minecraft:lapis_ore
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - forge:ingots/copper: janoeo:copper_ingot
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - minecolonies:reduceable_ingredient: janoeo:lead_ingot
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - minecolonies:blacksmith_product: janoeo:lead_ingot
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - forge:ores/iron: minecraft:iron_ore
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - forge:ores/emerald: minecraft:emerald_ore
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - forge:ingots/lead: janoeo:lead_ingot
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - minecolonies:blacksmith_ingredient: janoeo:lead_ingot
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - forge:ores/gold: minecraft:gold_ore
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - forge:ores/diamond: minecraft:diamond_ore
[21:56:54] [Server thread/INFO] [minecraft/DedicatedServer]: - forge:ingots/tin: janoeo:tin_ingot

```

As I then have to remove the five MineColonies entries so that only `forge:ingots/lead tag` is left. :)
```
[21:51:53] [Server thread/INFO] [minecraft/DedicatedServer]: [sandalle: Removed conversion with tag minecolonies:blacksmith_ingredient, if it exists]
[21:51:55] [Server thread/INFO] [minecraft/DedicatedServer]: [sandalle: Removed conversion with tag minecolonies:blacksmith_product, if it exists]
[21:51:59] [Server thread/INFO] [minecraft/DedicatedServer]: [sandalle: Removed conversion with tag minecolonies:reduceable_ingredient, if it exists]
[21:56:09] [Server thread/INFO] [minecraft/DedicatedServer]: [sandalle: Removed conversion with tag minecolonies:reduceable_product_excluded, if it exists]
[21:56:13] [Server thread/INFO] [minecraft/DedicatedServer]: [sandalle: Removed conversion with tag minecolonies:sawmill_ingredient_excluded, if it exists]
```

**Comment by sandalle** [comment]
Looks like Resourceful Bees does similar with ore blocks:
```
[22:04:30] [Server thread/INFO] [minecraft/DedicatedServer]: [sandalle: Added janoeo:lead_ore as the default conversion for the forge:ores/lead tag]
[22:04:30] [Server thread/INFO] [minecraft/DedicatedServer]: [sandalle: Added janoeo:lead_ore as the default conversion for the resourcefulbees:valid_apiary tag]
```

**Comment by MattDahEpic** [comment]
Fixed in 16.1.6, thanks for letting me know!

**Comment by sandalle** [comment]
Wow, thank you for the quick turn-around! autooredictconv-1.16.4-16.1.6.jar no longer adds MineColonies or Resourceful Bees entries when I run `/odc add` for an ingot or ore. :)
